### PR TITLE
Remove CentOS7 based images

### PR DESCRIPTION
**Pull request by zmiklank**
This PR also removes not needed cccp.yml and removes files from deprecated 1.18 nginx version.

### CODE REVIEW

**Comment by zmiklank** [comment]
[test-all]

**Comment by zmiklank** [comment]
[test-all]

**Comment by pkubatrh** [comment]
Im a bit confused here - did we not decide to only stop testing and pushing the images to quay.io? I would rather not remove all the sources before Centos 7 EOL.

**Comment by zmiklank** [comment]
> Im a bit confused here - did we not decide to only stop testing and pushing the images to quay.io? I would rather not remove all the sources before Centos 7 EOL.

Sure. I will add .exclude- files then and return the containerfiles back.

**Comment by zmiklank** [comment]
[test-all]

File: 1.20/README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 Nginx 1.20 server and a reverse proxy server container image
 ============================================================
 This container image includes Nginx 1.20 server and a reverse server for OpenShift and general usage.
-Users can choose between RHEL, CentOS, CentOS Stream and Fedora based images.
+Users can choose between RHEL, CentOS Stream and Fedora based images.
 The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
-the CentOS images are available in the [Quay.io](https://quay.io/organization/centos7),
```
Comment (pkubatrh) [inline 1.20/README.md:6]: I would keep these two lines - the images are still available to download and we do expect them to be used until the EOL of centos7.

**Comment by pkubatrh** [comment]
otherwise lgtm

**Pull request merged**